---a/net/minecraft/entity/boss/EntityDragonPart.java
+++b/net/minecraft/entity/boss/EntityDragonPart.java
@@ -5,39 +5,38 @@
 import net.minecraft.nbt.NBTTagCompound;
 import net.minecraft.util.DamageSource;
 
-public class EntityDragonPart extends Entity
-{
+public class EntityDragonPart extends Entity {
+
     public final IEntityMultiPart entityDragonObj;
     public final String partName;
     private static final String __OBFID = "CL_00001657";
 
-    public EntityDragonPart(IEntityMultiPart parent, String partName, float base, float sizeHeight)
-    {
+    public EntityDragonPart(IEntityMultiPart parent, String partName, float base, float sizeHeight) {
         super(parent.getWorld());
         this.setSize(base, sizeHeight);
         this.entityDragonObj = parent;
         this.partName = partName;
     }
 
-    protected void entityInit() {}
+    protected void entityInit() {
+    }
 
-    protected void readEntityFromNBT(NBTTagCompound tagCompund) {}
+    protected void readEntityFromNBT(NBTTagCompound tagCompund) {
+    }
 
-    protected void writeEntityToNBT(NBTTagCompound tagCompound) {}
+    protected void writeEntityToNBT(NBTTagCompound tagCompound) {
+    }
 
-    public boolean canBeCollidedWith()
-    {
+    public boolean canBeCollidedWith() {
         return true;
     }
 
-    public boolean attackEntityFrom(DamageSource source, float amount)
-    {
+    public boolean attackEntityFrom(DamageSource source, float amount) {
         return this.isEntityInvulnerable(source) ? false : this.entityDragonObj.attackEntityFromPart(this, source, amount);
     }
 
-    public boolean isEntityEqual(Entity entityIn)
-    {
+    public boolean isEntityEqual(Entity entityIn) {
         return this == entityIn || this.entityDragonObj == entityIn;
     }
-}
+}
